src/zenml/config/docker_settings.py

```diff
@@ -122,6 +122,12 @@ class DockerSettings(BaseSettings):
             this image.
             * If a custom `dockerfile` is specified for this settings
             object, this parent image will be ignored.
+        image_tag: [Optional] Docker image tag use for generated image.
+
+            Additional notes:
+            * This will override the default zenml tagging mechanism
+            using step.name or pipeline.name
+            * Make sure the tag you provide does not lead to image conflicts.
```

Comment:
Maybe this should be more something along the lines of:
`This tag will be used for all images built for the pipeline or step (depending on where you define your DockerSettings). If there are multiple such images, only one of them will keep the tag while the rest will be untagged.`

To give some more explanation?